### PR TITLE
test(olm): test role aggregation for aggregate apiservices

### PR DESCRIPTION
**Pull request by ecordell**
We were testing that CRD access was being aggregated to the `admin` cluster role; this adds an additional service to the namespace provided by an aggregated apiserver to test that as well.

### CODE REVIEW

File: test/e2e/operator_groups_e2e_test.go [hunk 1]
```diff
@@ -496,10 +497,82 @@ func TestOperatorGroupRoleAggregation(t *testing.T) {
...
+				Owned: owned,
+			},
+		},
+	}
+	csvB.SetName(depName)
+
+	// Create the APIService CSV
+	_, err = createCSV(t, c, crc, csvB, nsA, false, true)
+	require.NoError(t, err)
+	// defer cleanupCSV()
```
Comment (tkashem) [inline test/e2e/operator_groups_e2e_test.go]: We don't need to clean up the CSV?

Comment (ecordell) [reply test/e2e/operator_groups_e2e_test.go]: Whoops!

**Pull request merged**